constants/bad_data.php

```diff
@@ -19,3 +19,5 @@
 const NON_JOURNAL_BIBCODES = array('arXiv', 'gr.qc', 'hep.ex', 'hep.lat', 'hep.ph', 'hep.th', 
                                    'math.ph', 'math', 'nucl.ex', 'nucl.th', 'physics');
 const NON_PUBLISHERS = array('books.google', 'google books', 'google news', 'google.co', 'amazon.com', 'archive.org'); // Google Inc is a valid publisher, however.
+const BAD_ZOTERO_TITLES = array ('on jstor', 'Browse publications', 'Central Authentication Service',
```

Comment:
Can "on jstor" sometimes be appended to the end of an otherwise good title?  A better response there might be to strip the "on jstor" and keep the rest.

Reply:
If you get "on jstor" then the citoid stuff thought it was a website and almost everything is wrong.
Run it again later and you are good to go.